src/App.js

```diff
@@ -32,15 +33,69 @@ function App() {
       job: "github",
     },
   };
+  const [apiData, setApiData] = useState(UserData);
 
   const ContactIcons = [faLocationDot, faTwitter, faLink, faBuilding];
 
+  const searchHandler = async (username) => {
+    try {
+      const resUser = await fetch(`https://api.github.com/users/${username}`);
+      if (!resUser.ok) throw new Error("Problem getting user data");
+      console.log(resUser);
+      const data = await resUser.json();
+
+      const {
+        avatar_url,
+        name,
+        login,
+        created_at,
+        bio,
+        public_repos,
+        followers,
+        following,
+        location,
+        twitter_username,
+        blog,
+        company,
+      } = data;
+
+      setApiData({
+        profile: {
+          image: `${avatar_url}`,
+          name: `${name || "Not Available"}`,
+          handle: `${login || "Not Available"}`,
+          bio: `${bio || "This profile has no bio"}`,
+          date: `Joined ${
+            new Date(created_at).toLocaleString("en-US", {
+              day: "2-digit",
+              month: "short",
+              year: "numeric",
+            }) || "Not Available"
+          }`,
+        },
+        account: {
+          repo: `${public_repos || "-"}`,
+          followers: `${followers || "-"}`,
+          following: `${following || "-"}`,
+        },
+        contact: {
+          location: `${location || "Not Available"}`,
+          twitter: `${twitter_username || "Not Available"}`,
+          website: `${blog || "Not Available"}`,
+          job: `${company || "Not Available"}`,
+        },
+      });
+    } catch (err) {
+      console.error(`An error occurred: ${err}`);
```

Comment:
Add a line to setApiData to empty incase of an error as well. This will set the fields back to default values and signals to the user that a fetch was made but no data was returned so they will then check their input. Having no response change will leave them unsure as well of what to do. 

```suggestion
      setApiData({});
      console.error(`An error occurred: ${err}`);
```